keps/0007-skeleton-generator.md

```diff
@@ -0,0 +1,37 @@
+---
+kep-number: 7
+title: Skeleton generator
+authors:
+  - "@djannot"
+owners:
+  - TBD
+editor: TBD
+creation-date: 2019-04-01
+status: provisional
+---
+
+# Skeleton generator
+
+## Table of Contents
+
+  * [Table of Contents](#table-of-contents)
+  * [Summary](#summary)
+  * [Motivation](#motivation)
+
+Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
+
+## Summary
```

Comment:
I think a couple of CLI calls would help identify the functionality.  Will this just be an initial tool to get frameworks started, or be a tool for adding parameters/plans/etc

Reply:
At the beginning, I would say just to get frameworks started, but with all the plans (deploy, update, upgrade, ...).
And for each plan, at least a Phase, a Step and a Task.
Then, the user can delete the components he doesn't want to implement.